.eslintignore

```diff
@@ -1,5 +0,0 @@
-# Next.js
```

Comment:
why delete this na? I think we don't need to lint artifact or cache files

Reply:
Moved to `ignorePatterns` in `.eslintrc`